config/settings.yml

```diff
@@ -1,6 +1,9 @@
 ---
 :ems:
   :ems_redhat:
+    :default_product_version_to_api_version_regex_hash:
+      :'^3\..*$': ['3']
+      :'^4\..*$': ['3', '4']
```

Comment:
Would it be possible to avoid the Ruby syntax in the values? I understand that it would require additional parsing in the client, but I'd say that is acceptable. Ideally this should be just the version numbers, separated by spaces or commas.